.rubocop.yml

```diff
@@ -0,0 +1,26 @@
+inherit_from: .rubocop_todo.yml
+
+AllCops:
+  Exclude:
+    - vendor/**/*
+    - gemfiles/**/*
+
+Metrics/LineLength:
+  Max: 140
+
+# Can't do without a little table-flip there ;)
+Style/AsciiComments:
+  Exclude:
+    - 'test/scope_test.rb'
```

Comment:
Hmm, I'm not saying that isn't true, but which words are you referring to that could appear in a code comment? Unless you really want it to be _naïve_ instead of _naive_ or _à propos_ instead of _apropos_, there are, I'd argue – at least appearing in source code – indeed very few English words with accents.

I'd also say it also makes sense for smaller projects that are open to public contributions, like this one.

But I'll change it, if you insist.